Balance/View Controllers/Tabs/BalanceViewController.swift

```diff
@@ -181,7 +182,7 @@ class BalanceViewController: UITableViewController {
         case .erc20:
             // Check if there are any tokens
             for wallet in ethereumWallets {
-                if let count = wallet.tokens?.count, count > 0 {
+                if let count = wallet.nonZeroTokens?.count, count > 0 {
```

Comment:
actually I'm about to work on cell expansion, so we still want the zero tokens (or less than $10 worth tokens), they'll just be hidden until you expand the cell